Documentation/learning/api.md

```diff
@@ -47,7 +47,7 @@ message ResponseHeader {
 
 An application may read the Cluster_ID (Member_ID) field to ensure it is communicating with the intended cluster (member).
```

Comment:
Cluster_ID and Member_ID should be emphasized as `Cluster_ID` and `Member_ID` to be consistent with other fields (e.g. `Revision` or `Raft_Term`). Also I think since they are two different field, re-writing "Cluster_ID (Member_ID)" as "`Cluster_ID` or `Member_ID`" would be proper. Thanks!